apk-generator/v2/app/templates/index.html

```diff
@@ -25,7 +25,7 @@
     <form class="col-md-offset-4 col-xs-offset-2 col-xs-8 col-md-4 form-group generator_form" method="post" id="form"
           name="form" role="form" enctype="multipart/form-data" autocomplete="off">
         <label for="email">Email</label>
-        <input type="email" class="form-control" id="email" name="email" required>
+        <input type="email" class="form-control" id="email" name="email" required autofocus autocomplete="on">
```

Comment:
autocomplete="on" is not required. Just remove `autocomplete="off"` from form tag.